server/routes.js

```diff
@@ -29,4 +39,41 @@ router.put('/api/addtopack', addUserToPack);
 // Expects a pack_name and array of user ids in the req body
 router.put('/api/createpack', createNewPackAndAdd);
 
+router.post('/getAllPostsFromSpecificPack', (req, res) => {
```

Comment:
routes should have /api/getAll...

Reply:
understood, making that change now

Reply:
this should be a get request as well I think? Seems like you're requesting info and not changing anything

Same for getUserPacks and the other gets below